snap/snapcraft.yaml

```diff
@@ -492,34 +492,6 @@ parts:
   # from edgex-go which at this point will fail
```

Comment:
on an unrelated note, this comment can be cleaned up, we don't actually build go anymore so this doesn't apply

Reply:
done